code/game/sound.dm

```diff
@@ -98,28 +85,40 @@
 			-1000, 1.0, \
 			0, 1.0, 1.0, 1.0, 1.0, 7)
 
-	if(!is_global) S.environment = 2
+	if(!is_global)
+		S.environment = 2
+
 	SEND_SOUND(src, S)
 
 
-/mob/living/playsound_local(turf/turf_source, soundin, vol, vary, frequency, falloff, is_global)
+/mob/living/playsound_local(turf/turf_source, soundin, vol, vary, frequency, falloff, is_global, channel = 0, pressure_affected = TRUE, sound/S)
```

Comment:
Since you've removed the pressure code, `pressure_affected` should be removed too. Also, why two variables for the sound? soundin vs S

Reply:
`playsound()` generates the `var/sound/S` which is sent to each recipient's `playsound_local()` instead of the latter generating it for every recipient.
Though there are situations (such as hallucinations) where `playsound_local()` is used standalone, and thus soundin is still necessary for them to generate the sound datum.
I assume `istype()` would be detrimental to its performance, and thus it's better to leave both vars.

Btw thanks for warning me about the pressure_affected var leftover.